lib/stripe/api_operations/request.rb

```diff
@@ -5,6 +5,8 @@ module ClassMethods
         OPTS_KEYS_TO_PERSIST = Set[:api_key, :api_base, :client, :stripe_account, :stripe_version]
 
         def request(method, url, params={}, opts={})
+          Util.validate_params!(params)
```

Comment:
Given that we're only going to be using this helper in one place, lets move it back to `request.rb` okay?